node/node.go

```diff
@@ -125,25 +125,31 @@ const (
 )
 
 func GetCommand() *cobra.Command {
+	conf := config.MainnetConfig()
+	var configPath *string
 	c := &cobra.Command{
 		Use:   "node",
 		Short: "start node",
 		Run: func(c *cobra.Command, args []string) {
```

Comment:
If we change this to use `RunE` instead of `Run` we can return early from the function. The returned error will be printed to `Stderr` and the program will exit with status 1 (as defined in `main.go`)

```suggestion
		RunE: func(c *cobra.Command, args []string) error {
```